### PR TITLE
Use aldryn translation tools

### PR DESCRIPTION
**Pull request by mikek**
Use aldryn translation tools to generate slugs.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+1.0.2 (yyyy-mm-dd)
```
Comment (intellectronica) [inline CHANGELOG.rst]: is there some magic that turns this into the actual date?

**Comment by intellectronica** [comment]
:+1: but see comment on changelog

File: CHANGELOG.rst (hunk 1 above)
Comment (mikek) [inline CHANGELOG.rst]: No magic. could just put 'Next' as a the version as well.

Comment (intellectronica) [inline CHANGELOG.rst]: Ah got it

**Pull request merged**